tests/acceptance/new-test.js

```diff
@@ -101,6 +101,104 @@ describe('Acceptance: ember new', function () {
     expect(file('app/templates/application.hbs')).to.contain('Welcome to Ember');
   });
 
+  // [WIP] ember new --lang flag
+  // -------------------------------
+  // Good: Default
+  it('ember new without --lang flag (default) has no lang attribute in index.html', async function () {
+    await ember(['new', 'foo', '--skip-npm', '--skip-bower', '--skip-git']);
+    expect(file('app/index.html')).to.contain('<html>');
+  });
+
```

Comment:
I'd prefer to account for this in the main blueprinting tests for default output of `new`/`addon` output here:

https://github.com/ember-cli/ember-cli/blob/5237dc7f67e766c8da1a56e329b6c91369c0c34c/tests/acceptance/new-test.js#L439-L460


```suggestion
```

Reply:
Relocated in 5ca7693f 👍